docs/integrations/builtin/credentials/homeassistant.md

```diff
@@ -27,10 +22,18 @@ Refer to [Home Assistant's API documentation](https://developers.home-assistant.
 
 ## Using API access token
 
-To configure this credential, you'll need:
+To configure this credential, you'll need to [Install](https://www.home-assistant.io/installation/) Home Assistant, create a [Home Assistant](https://www.home-assistant.io/getting-started/onboarding) account, and have:
+
+- Your **Host**
+- The **Port**
+- A Long-Lived **Access Token**
+
+To generate an access token and set up the credential:
 
-- Your **Host**: The URL or IP Address of your host. Refer to the [Rest API documentation](https://developers.home-assistant.io/docs/api/rest/){:target=_blank .external-link} for more information.
-- The **Port**: Home Assistant defaults to `8123`. Only change this if you've explicitly changed the port used.
-- **SSL**: If you've enabled SSL in Home Assistant in the [config.yml map key](https://developers.home-assistant.io/docs/add-ons/configuration/?_highlight=ssl#add-on-configuration){:target=_blank .external-link}, toggle this setting on.
-- An **Access Token**: n8n expects a [Long-Lived Access Token](https://developers.home-assistant.io/docs/auth_api#long-lived-access-token){:target=_blank .external-link}), which you can view in Home Assistant through your profile. Refer to the [Rest API documentation introduction](https://developers.home-assistant.io/docs/api/rest/){:target=_blank .external-link} for instructions on generating one.
+1. To generate your **Access Token**, log in to Home Assistant and open your [User profile](https://my.home-assistant.io/redirect/profile){:target=_blank .external-link}.
+2. In the **Long-Lived Access Tokens** section, generate a new token.
+3. Copy this token and enter it in n8n as your **Access Token**.
+4. Enter the URL or IP address of your Home Assistant **Host**, for example `https://your.awesome.home`.
+5. For the **Port**, Home Assistant defaults to `8123`. If you've configured Home Assistant to use a different port, enter the correct port.
+6. If you've enabled SSL in Home Assistant in the [config.yml map key](https://developers.home-assistant.io/docs/add-ons/configuration/?_highlight=ssl#add-on-configuration){:target=_blank .external-link}, turn on the **SSL** toggle in n8n. If you're not sure, it's best to turn this setting on if your **Host** begins with `https:` instead of `http`.
```

Comment:
```suggestion
6. If you've enabled SSL in Home Assistant in the [config.yml map key](https://developers.home-assistant.io/docs/add-ons/configuration/?_highlight=ssl#add-on-configuration){:target=_blank .external-link}, turn on the **SSL** toggle in n8n. If you're not sure, it's best to turn this setting on if your **Host** begins with `https:` instead of `http:`.
```